code/game/machinery/alarm.dm

```diff
@@ -973,7 +973,7 @@ FIRE ALARM
 	return src.alarm()
 
 /obj/machinery/firealarm/CtrlClick(var/mob/user)
-	if(user.incapacitated() || (!Adjacent(user) && !issilicon(user) && !(M_TK in usr.mutations)))
+	if(user.incapacitated() || (!Adjacent(user) && !issilicon(user) && !isAI(user) && !(M_TK in usr.mutations)))
```

Comment:
i'm pretty sure AIs are silicons

Reply:
They're differentiated, @SonixApache reported it.

Reply:
Is this tested? Because yes, AIs *are* silicons
![image](https://user-images.githubusercontent.com/9827785/53046953-41ae7300-345f-11e9-804b-0dc79274a31d.png)

Reply:
Give me collab so I can give this the `100%`
The logic is wrong, it checks if you're not incapacitated, and you're adjacent AND neither an AI nor have telekinesis
I think it would be better if you did
```dm
var/can_remotely_access = issilicon(user) || (M_TK in user.mutations)
if (user.incapacitated() || (!Ajdacent(user) && !can_remotely_access)
     // ...
```
However I'm not sure the logic is wrong because it's equivalent to what is currently in the code.
Maybe it's `Adjacent(user)` ?

Reply:
By the way it should also use `in_range()` instead of the direct `Adjacent()` check and TK check

Reply:
A fix is coming through.
AI Eye is different from AI silicon. Trust me, I tried and tested, and there's even an in-game report that it's not possible to ctrl-click as an AI. It's because the user is counted as the AI eye, not the AI mob itself

Reply:
In that case, this still wouldn't work, because the AI eye fails `isAI()`

Reply:
Also the real problem is that AI has its own snowflake proc called AICtrlClick for some godforsaken reason